Careless mistake in commpath PR

A reminder that I need to be compiling with `-Wall -Wextra -Werror`.
```
  CC       ../../utils/fileutil.lo
In file included from /home/slurmuser/Spindle/src/server/../logging/spindle_debug.h:30,
                 from /home/slurmuser/Spindle/src/server/auditserver/cleanup_proc.cc:19:
/home/slurmuser/Spindle/src/server/auditserver/cleanup_proc.cc: In function 'void rmDirSet(const std::set<std::__cxx11::basic_string<char> >&, const char*, const char*)':
/home/slurmuser/Spindle/src/server/../logging/spindle_logc.h:91:42: warning: format '%s' expects a matching 'char*' argument [-Wformat=]
   91 |          fprintf(spindle_debug_output_f, "[%s.%d@%s:%u] - ERROR: "      \
/home/slurmuser/Spindle/src/server/auditserver/cleanup_proc.cc:59:8: note: in expansion of macro 'err_printf'
   59 |        err_printf( "cachepath (%s) and/or commpath (%s) is NULL.  Unable to cleanup files.\n" );
      |        ^~~~~~~~~~
/home/slurmuser/Spindle/src/server/auditserver/cleanup_proc.cc:59:33: note: format string is defined here
   59 |        err_printf( "cachepath (%s) and/or commpath (%s) is NULL.  Unable to cleanup files.\n" );
      |                                ~^
      |                                 |
      |                                 char*
/home/slurmuser/Spindle/src/server/../logging/spindle_logc.h:91:42: warning: format '%s' expects a matching 'char*' argument [-Wformat=]
   91 |          fprintf(spindle_debug_output_f, "[%s.%d@%s:%u] - ERROR: "      \
/home/slurmuser/Spindle/src/server/auditserver/cleanup_proc.cc:59:8: note: in expansion of macro 'err_printf'
   59 |        err_printf( "cachepath (%s) and/or commpath (%s) is NULL.  Unable to cleanup files.\n" );
      |        ^~~~~~~~~~
/home/slurmuser/Spindle/src/server/auditserver/cleanup_proc.cc:59:54: note: format string is defined here
   59 |        err_printf( "cachepath (%s) and/or commpath (%s) is NULL.  Unable to cleanup files.\n" );
      |                                                     ~^
      |                                                      |
      |                                                      char*
  CXXLD    libserverbase.la
```

--- a/src/client/beboot/spindle_bootstrap.c
+++ b/src/client/beboot/spindle_bootstrap.c
@@ -148,6 +148,7 @@ static int setup_environment()
          free(preload);
       }
    }
+   return 0;
 }
 
 static int parse_cmdline(int argc, char *argv[])

--- a/src/client/client/intercept_readlink.c
+++ b/src/client/client/intercept_readlink.c
@@ -27,6 +27,7 @@ Place, Suite 330, Boston, MA 02111-1307 USA
 #include "client_api.h"
 #include "spindle_launch.h"
 #include "should_intercept.h"
+#include "ccwarns.h"
 
 ssize_t (*orig_readlink)(const char *path, char *buf, size_t bufsiz);
 ssize_t (*orig_readlinkat)(int dirfd, const char *pathname, char *buf, size_t bufsiz);
@@ -114,8 +115,10 @@ ssize_t readlink_wrapper(const char *path, char *buf, size_t bufsiz)
    }
 
    len = strlen(resultpath);
+GCC7_DISABLE_WARNING("-Wsign-compare");
    if (len > bufsiz)
       len = bufsiz;
+GCC7_ENABLE_WARNING;
    memcpy(buf, resultpath, len);
    debug_printf2("spindle readlink translated %s to %.*s with len %zu\n", path, (int)len, buf, len);
    return len;

--- a/src/server/auditserver/cleanup_proc.cc
+++ b/src/server/auditserver/cleanup_proc.cc
@@ -56,7 +56,8 @@ static void rmDirSet(const set<string> &dirs, const char *cachepath, const char 
    string path_sep("/");
    if( !cachepath || !commpath ){
        // Should never happen.
-       err_printf( "cachepath (%s) and/or commpath (%s) is NULL.  Unable to cleanup files.\n" );
+       err_printf( "cachepath (%s) and/or commpath (%s) is NULL.  Unable to cleanup files.\n",
+              cachepath, commpath );
        return;
    }
    size_t cachepath_len = strlen(cachepath);

